classes/xt_rcar.bbclass

```diff
@@ -0,0 +1,8 @@
+rcar_unpack_proprietary_graphics() {
```

Comment:
Rename it to i.e. rcar_unpack_evaproprietary to correspondent to the packages content as well as follow Renesas'es style